### ISSUE TITLE
Login/sign up on other devices not working

### ISSUE THREAD

**Issue by Hassankhd** [dalenguyen/firebase-wordpress-plugin#173]
Hi Dale!

I just launched my wordpress site, and everything related to firebase was working just fine whenever I previewed the homepage and tested the whole website. However, I just opened the site on my iPhone and as soon as I come to log in with an existing user, no log in happens. When I create a new user, the user is created but nothing else happens (i.e. their extra info isn't saved in the database and the user isn't logged in). I must also mention 2 things:

1) All of this was working just as I wanted it to prior to launching the site

2) I was logging in as another user / signing up as a new user on my phone as I was already logged in on my laptop, as I wanted to test the video chat element of the website.

Could it be that Firebase isn't letting me log in 2+ users at a time? I'm pretty sure my code is correct since everything was going the way I wanted. I would really appreciate your help. Thanks in advance!

Hassan

**Comment by Hassankhd** [comment]
Update: I also just tried signing up as a new user on my iPad, and this is what I got:

"A network error (such as timeout, interrupted connection or unreachable host) has occurred."

**Comment by dalenguyen** [comment]
Hi @Hassankhd, firebase should allow any users to sign In / sign up properly as long as it's not the same session / browser. The iPad could be a network-related issue. What site are you testing?

**Comment by Hassankhd** [comment]
Hey Dale, I'm really sorry for the late reply I had some exams to do this week. The site in question is jobseeqr.co

**Comment by Hassankhd** [comment]
Update: I just fixed the issue! Turns out Safari does not recognize a button in HTML unless you type it in as <button **type="button"**></button>. I just fixed the issue and thankfully my site is officially up and running! I really appreciate all the help you have given me with previous issues and thank you for making such an amazing plug-in! I will definitely come back should future issues arise while I test the site with a few friends of mine, but until then, thanks again for your help!
